manage.py

```diff
@@ -1,4 +1,4 @@
-from goodpackagenamehere.control import cli
+from ner_trainer.control import cli
```

Comment:
Yep, actually I've changed to ner_trainer also as temporary name, but which looks more production-ready comparing to goodpackagenamehere :) So, if there are decision already on name, I will change to new name, could also change to goodpackagenamehere if needed